Verified publishers: add recommendation for several members (#5467)
dart-lang/site-www: src/tools/pub/publishing.md

   * When the verification flow is complete, return to step 4.

> {{site.alert.note}} We strongly recommend you invite other members of your
>   organization as members to the verified publisher. This ensures that your
>   organization retains access to the publisher when you are not available.
>   {{site.alert.end}}

Reviewer: ```suggestion
{{site.alert.tip}}
  We strongly recommend you invite other members of your
  organization to be members of the verified publisher.
  This helps ensure that your organization retains access to
  the publisher when you are not available.
{{site.alert.end}}
```
